server/api-util/lineItems.js

```diff
@@ -140,18 +155,23 @@ exports.transactionLineItems = (listing, orderData, providerCommission, customer
   const quantityAndExtraLineItems =
     unitType === 'item'
       ? getItemQuantityAndLineItems(orderData, publicData, currency)
-      : unitType === 'hour'
+      : unitType === 'hour' && !orderData.seats
       ? getHourQuantityAndLineItems(orderData)
-      : ['day', 'night'].includes(unitType)
+      : unitType === 'hour'
+      ? getHoursWithSeatsAndLineItems(orderData)
+      : ['day', 'night'].includes(unitType) && !orderData.seats
       ? getDateRangeQuantityAndLineItems(orderData, code)
+      : ['day', 'night'].includes(unitType)
+      ? getDateRangeWithSeatsAndLineItems(orderData, code)
       : {};
```

Comment:
It feels strange to use negation (!orderData.seats) in this ternary. Especially, since the name has `HoursWithSeats` / `DateRangeWithSeats`

How about:
```js
    unitType === 'item'
      ? getItemQuantityAndLineItems(orderData, publicData, currency)
      : unitType === 'hour' && orderData.seats
      ? getHoursWithSeatsAndLineItems(orderData)
      : unitType === 'hour'
      ? getHourQuantityAndLineItems(orderData)
      : ['day', 'night'].includes(unitType) && orderData.seats
      ? getDateRangeWithSeatsAndLineItems(orderData, code)
      : ['day', 'night'].includes(unitType)
      ? getDateRangeQuantityAndLineItems(orderData, code)
      : {};
```

Reply:
This does not work yet.